### DISCUSSION TITLE
`TAB` support for `magit-repolist-mode`

### DISCUSSION THREAD

**Discussion by yliceee** [magit/magit#5043] [Show and tell]
Hi all,

I often find myself using `magit-list-repositories` to check the status of a lot of repos scattered throughout my system. Since some of my systems have slow physical disks, and I have a lot of repos, it can be slow to startup. I thought it would be helpful to add a `TAB` action like in the org-agenda which opens the status buffer of the repo under point in a new window. Here is a scratch attempt:
```lisp
(defun my/magit-repolist-status (&optional _button)
  "Show the status for the repository at point."
  (interactive)
  (if-let ((id (tabulated-list-get-id)))
      (progn
        (switch-to-buffer-other-window nil)
        (magit-status-setup-buffer (expand-file-name id))
        )
    (user-error "There is no repository at point")))
```
There is a minor bug that if you run the command twice on the same repo, it will "skip" a section. Maybe someone will also like this function.

**Comment by tarsius** [comment]
I'm going through my `idea` branch which contains a ton of draft commits, including one for the point you are raising. Since then I have learned about `C-x 4 4`, which can be used (pretty much) any command that displays a buffer, and instructs it to display that in a another window. There's also `C-x 5 5`, which uses a new frame.
